docs/tutorials/get-started-with-ngo.md

```diff
@@ -0,0 +1,619 @@
+
+3. Select **NetworkManager**, then select **Add Component** from the **Inspector** tab.
+
+4. Select **Netcode** > **NetworkManager** from the component list.
+
+5. Select **UnityTransport** as the **Network Transport**.
+6. Save the scene by selecting **File** > **Save**.
+
+### Create an object to spawn for each connected player
+
+This section guides you through creating an object that spawns for each connected player.
+
+1. In the Unity Editor, right-click within the **Hierarchy** tab, then select **3D Object** > **Capsule**.
+2. Name the Capsule Object **Player**.
+3. With Player selected, add a `NetworkObject` component in the **Inspector** tab by selecting **Add Component** > **NetworkObject**.
+4. Right-click within the **Assets** folder under the **Project** tab, then select **Create** > **Folder**.
+5. Name the folder **Prefabs**.
+6. Make the **Player** object you created earlier into a prefab by dragging it from the **Hierarchy** tab into the **Prefabs** folder.
+
+7. Delete the Player from the scene by selecting the Player capsule within the **Scene** tab, then pressing the **Delete** key.
+
+:::tip
+You can remove the `Player` object from the scene because you assign this network prefab to the Player Prefab property in the `NetworkManager` component. The library doesn't support defining a player object as an in-scene placed `NetworkObject`.
+:::
+
+8. Select **NetworkManager**.
+9. With `NetworkManager` selected, locate the **PlayerPrefabs** field in the **Inspector** tab.
+
+10. Drag the **Player** prefab from the **Project** tab into the **PlayerPrefab** slot you created in the **Inspector** tab.
+
+:::important
+
+When you drop the prefab into the **PlayerPrefab** slot, you are telling the library that when a client connects to the game, it should automatically spawn this prefab as the character for the connecting client. NGO won’t spawn a player object if you do not have any prefab set as the **PlayerPrefab**.
+:::
+
+11. Add a 3D Plane (centered at 0,0,0) to the scene by right-clicking in the **Hierarchy** tab, then selecting **3D Object** > **Plane**.
+
+:::note
+
+Adding the Plane adds a visual reference point to visualize the Player prefab’s position, but it isn’t necessary.
+
+:::
+
+12. Save the scene by selecting **File** > **Save**.
+
+### Add your scene to the build
+
+This section guides you through adding your scene to the build.
+
+The **Enable Scene Management** for the `NetworkManager` setting allows the server to control which scenes should load for the clients. However, you must add the current scene to the build to enter Play mode.
+
+:::note
+
+The **Enable Scene Management** option for the `NetworkManager` is enabled by default.
+
+:::
+
+1. Open the Build Settings window by selecting **File** > **Build Settings**.
+2. Select **Add Open Scenes**.
+
+You should see **Scenes/SampleScene** listed under **Scenes In Build**. You can close the Build Settings window.
+
```

Comment:
Could be interesting to add a small bit here telling users to try out this build at this point with the command line helper to see how the player prefab is spawned when the client connects, or is that too early?

Reply:
It might be too early. It'd be neat, but it might clutter the content a little bit. Maybe we could add it as a collapsed note?

Reply:
It could also be a single sentence, saying that testing is now possible with the command line helper and describing what it should look like (just having the player prefab spawn when connecting a client to the server, or starting as a host)